docs/get-started/developers/tokens.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: layout-with-subnav.njk
+title: Tokens
+heading: Developers
+tags:
+  - developers
+permalink: /get-started/developers/tokens/index.html
+order: 30
+bodyClasses: element-docs
+---
+
+## How to install tokens
+
+Run the following git command to install RHDS tokens:
+
+<rh-code-block>
+npm i @rhds/tokens
+</rh-code-block>
+
+## Usage
+
+We use [style-dictionary](https://amzn.github.io/style-dictionary/) to transform our tokens into multiple formats and helpers.
+
+### Import global CSS
+
+Apply defaults to the document root by importing the global stylesheet:
+
+<rh-code-block>
+    <script type="text/html">
+<link rel="stylesheet" href="/url/to/@rhds/tokens/css/global.css">
+<style>
+  :is(h1, h2, h3, h4, h5, h6) {
+    font-family: var(--rh-font-family-heading);
+  }
+</style>
+  </script>
+</rh-code-block>
+
+### Reset the shadowroot
+
+Reset a component's styles (preventing inheritance) by adding resetStyles to its static Constructible Style Sheet list:
+
+<rh-code-block>
+  <script type="text/sample-javascript">
+import { resetStyles } from '@rhds/tokens/css/reset.css.js';
+import style from './rh-jazz-hands.css';
+
+@customElement('rh-jazz-hands')
+export class RhJazzHands extends LitElement {
+  static readonly styles = [resetStyles, style];
```

Comment:
Brian asked:

Should this be `[...resetStyles, style];`?

Reply:
yes that is correct, you want to add the spread operator here.

The `...` is a spread operator, and is used to include all elements from an iterable (like an array) into another iterable.